login-workflow/src/new-architecture/screens/ResetPasswordScreen/ResetPasswordScreen.tsx

```diff
@@ -66,13 +81,23 @@ export const ResetPasswordScreen: React.FC<ResetPasswordScreenProps> = (props) =
         [setPasswordInput, setConfirmInput]
     );
 
+    const errorConfig: ErrorManagerProps = {
+        mode: 'dialog',
+        error: errorMsg,
+        dialogConfig: {
+            title: t('bluiCommon:MESSAGES.ERROR'),
+            dismissLabel: t('bluiCommon:ACTIONS.OKAY'),
+        },
+        onClose: (): void => { setErrorMsg(''); navigate(routeConfig.LOGIN)},
+    };
+
     const {
         WorkflowCardBaseProps,
         WorkflowCardHeaderProps,
         WorkflowCardInstructionProps,
         WorkflowCardActionsProps,
         PasswordProps,
-        errorDisplayConfig = errorManagerConfig,
+        errorDisplayConfig = errorConfig,
```

Comment:
We need to use the errorManagerConfig as that holds the global scope of the error config. If the user modifies the config on the auth provider or the reset password component itself it should override our defaults for error state.